requirements.txt

```diff
@@ -0,0 +1,42 @@
+bleach==6.0.0
```

Comment:
Are all these dependencies required? Did you create an empty environment and install your dependency to make sure this requirements.txt is minimal>

Reply:
Yeah I did, but maybe something extra got added I can repeat it to make sure. A lot of extra dependencies got installed when I installed Panel and hvplot. I also removed some libraries I installed earlier but didn't need to use anymore so maybe those didn't actually get removed.